ftp/src/main/scala/org/apache/pekko/stream/connectors/ftp/model.scala

```diff
@@ -247,6 +252,7 @@ final class FtpsSettings private (
     s"credentials=$credentials," +
     s"binary=$binary," +
     s"passiveMode=$passiveMode," +
+    s"useFtpsImplicit=$useFtpsImplicit," +
     s"autodetectUTF8=$autodetectUTF8" +
```

Comment:
"autodetectUTF8=$autodetectUTF8" should be
"autodetectUTF8=$autodetectUTF8,"

Reply:
a pre-existing issue - but I've pushed a fix to this PR